tests/functional/build-remote-with-mounted-ssh-ng.sh

```diff
@@ -5,17 +5,25 @@ source common.sh
 requireSandboxSupport
 [[ $busybox =~ busybox ]] || skipTest "no busybox"
 
+# An example of a command that uses the store only for its settings, to
+# make sure we catch needing the XP feature early.
+touch "$TEST_ROOT/foo"
+expect 1 nix --store 'ssh-ng://localhost?mounted=%7B%7D' store add "$TEST_ROOT/foo" --dry-run
```

Comment:
This may fail for the wrong reason. Please grep the error.
Is this the right place for this test?